src/style.scss

```diff
@@ -136,11 +136,12 @@ svg {
 :host([label]) {
   background-color: transparent;
   border-color: var(--auro-color-border-primary-on-light);
-  color: var(--auro-color-text-secondary-on-light);	
-  padding: 0.375rem 0.5625rem 0.1875rem 0.5625rem;
+  color: var(--auro-color-text-secondary-on-light);
+  font-size: var(--auro-text-body-size-xs);
+  padding: 4px 6px;
```

Comment:
Why is this moved to hard pixels? The badge supports the `fixed` feature. 

Please move back to the REM values.